lambdas/data-migration1/src/collections.ts

```diff
@@ -47,14 +47,14 @@ export const migrateCollectionRecord = async (
     version: dynamoRecord.version,
     process: dynamoRecord.process,
     url_path: dynamoRecord.url_path,
-    duplicateHandling: dynamoRecord.duplicateHandling,
-    granuleIdValidationRegex: dynamoRecord.granuleId,
-    granuleIdExtractionRegex: dynamoRecord.granuleIdExtraction,
+    duplicate_handling: dynamoRecord.duplicateHandling,
```

Comment:
I wasn't sure where to leave this, and it's not *directly* relevant to this PR, but as we go into writing 'writes' tickets and need to write translations like these we may want to consider consolidating them.